create-login-profile.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+
+const readline = require("readline");
+const child_process = require("child_process");
+
+const puppeteer = require("puppeteer-core");
+const yargs = require("yargs");
+
+function cliOpts() {
+  return {
+    "url": {
+      describe: "The URL of the login page",
+      type: "string",
+      demandOption: true,
+    },
+
+    "user": {
+      describe: "The username for the login. If not specified, will be prompted",
+    },
+
+    "password": {
+      describe: "The password for the login. If not specified, will be prompted (recommended)",
+    },
+
+    "filename": {
+      describe: "The filename for the profile tarball",
+      default: "/output/profile.tar.gz",
+    },
+
+    "debugScreenshot": {
+      describe: "If specified, take a screenshot after login and save as this filename"
+    },
+
+    "headless": {
+      describe: "Run in headless mode, otherwise start xvfb",
+      type: "boolean",
+      default: false,
+    },
+  }
+}
+
+
+
+async function main() {
+  const params = yargs
+    .usage("browsertrix-crawler profile [options]")
+    .option(cliOpts())
+    .argv;
+
+  if (!params.headless) {
+    console.log("Launching XVFB");
+    child_process.spawn("Xvfb", [
+      process.env.DISPLAY,
+      "-listen",
+      "tcp",
+      "-screen",
+      "0",
+      process.env.GEOMETRY,
+      "-ac",
+      "+extension",
+      "RANDR"
+    ]);
+  }
+
+  //await new Promise(resolve => setTimeout(resolve, 2000));
```

Comment:
Why is this commented out?

Reply:
Was just testing something during development and thought a delay was needed there, but that turned out to not be needed, so should probably just remove this.